app/models/rdv.rb

```diff
@@ -299,8 +304,17 @@ def soft_delete
     true
   end
 
+  def update_users_count
```

Comment:
Est-ce qu'il y a une raison pour ne pas utiliser un counter cache et laisser ce travail se faire automatiquement ? 
https://guides.rubyonrails.org/association_basics.html#options-for-belongs-to-counter-cache

Reply:
Alors en fait on doit compter les rdvs_users qui ne sont pas cancel (via le scope .not_canceled) https://github.com/betagouv/rdv-solidarites.fr/blob/fbcbb94cb03d3adcbbc9d2f0002609dffd95bebd/app/models/rdv.rb#L308 
Le counter cache intégré à rails ne permet pas de conditionner. Il compte toutes les associations sans distinction. Du coup j'ai du passer par une methode et des hook dédiés. Il y a aussi la possibilité d'utiliser une gem pour cela (counter_culture) mais vu l'usage j'ai préféré faire comme ca.

Reply:
OK compris, merci pour l'explication !